filebeat/crawler/prospector.go

```diff
@@ -97,35 +100,89 @@ func (p *Prospector) Run(wg *sync.WaitGroup) {
 				return
 			case event := <-p.harvesterChan:
```

Comment:
all harvesters pushing to harvesterChan? Why not of ProspectorStdin push to p.spoolerChan directly. This go-routine seem to be specific to ProspectorLog